_posts/2024-04-15-video-segmentation.md

```diff
@@ -0,0 +1,176 @@
+---
+layout: post
+title: "Exploring Transformer-Based Methods in Video Instance Segmentation"
+subtitle: "A comparison of the latest research and innovations in Video Instance Segmentation"
+description: "Get a detailed overview of the transformative impact of transformer models on video instance segmentation. Learn about the evolution of VIS methods, the role of datasets like YouTube-VIS, and the latest research trends that are defining the future of video processing technologies. This includes detailed comparisons among leading models like VisTR, IFC, and TeViT."
+image: "/assets/images/fullsize/posts/2024-04-15-video-segmentation/thumbnail.jpg"
+selected: y
+mathjax: n
+---
+
+I had recently visited the most recent literature on Video Segmentation and I found surpised about how lost I was. If you are related with it you probably are familiar with terms such as SVOS, UVOS, VIS, Zero-Shot, One-Shot, etc. If you are not then you will probably find yourself as lost as I was a few weeks ago.
+
+My main intention was to focus solely on the specific topic I was interested in. This is Video instance Segmentation (VIS) which extends the image instance segmentation task from the image domain to the video domain. The goal is to segment object instances from a predefined category set in videos and associate the instance identities across frames. It can also be seen as a combination of instance segmentation and object tracking in videos.
```

Comment:
```suggestion
My main intention was to focus solely on the specific topic I was interested in. This is Video instance Segmentation (VIS), which extends the image instance segmentation task from the image domain to the video domain. The goal is to segment object instances from a predefined category set in videos, then associate the instance identities across frames. It can also be seen as a combination of instance segmentation and object tracking in videos.
```